.github/workflows/test-redis-enterprise.yml

```diff
@@ -0,0 +1,66 @@
+name: RE Tests
+
+on:
+  push:
+    branches: [master, v9]
```

Comment:
IMHO for now master only.. We shouldn't go back to v9 branches (yet). But yes, we will future them